src/analyzer/protocol/syslog/syslog.evt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022 by the Zeek Project. See LICENSE for details.
+
+protocol analyzer Syslog over UDP:
```

Comment:
It took me a bit to understand why the service in `conn.log` doesn't have `spicy_syslog` and instead just `syslog` that's been showing up for other Spicy analyzers in quite a few places. It seems that is "just" because this line isn't `protocol analyzer spicy::Syslog`. 

Rather unrelated to this branch, but I wonder if we should update the package-template for protocol/packet/file analyzers to remove the usage of the `spicy::` prefix in the generated .evt files, too. The appearance of `spicy_` in the conn.log has come up elsewhere as not useful/appropriate or exposing an implementation detail.

I can start/create an issue over there as well.